packages/ui/src/molecules/PaymentMethods/PaymentMethods.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import Label from '../../atoms/Label'
+
+export interface PaymentMethodsProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Title of the payment methods section (e.g.: "Payment methods",
+   * "Accepted Cards").
+   */
+  title?: ReactNode
+  /**
+   * Children will receive the flags to be displayed in the payment
+   * methods section (e.g.:, visa, mastercard, etc).
+   */
+  children: ReactNode
+}
+
+const PaymentMethods = ({
+  testId = 'store-payment-methods',
+  title,
+  children,
+}: PaymentMethodsProps & { children: ReactNode }) => {
+  return (
+    <div data-store-payment-methods data-testid={testId}>
+      <Label>{title}</Label>
```

Comment:
```suggestion
      {title && <Label>{title}</Label>}
```

Reply:
```suggestion
      <Label data-payment-method-title>{title}</Label>
```

Reply:
Is it okay to use the [`label` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) outside of a form/input context?

Reply:
Hey @filipewl, good question. Labels aren't only used for [`Form-Associated`](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#form-associated_content) elements as you can see on the link you shared. Also, the aria-label or label is used to help assistive technology name things, according to https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html

> For sighted users, the context and visual appearance of an element can provide sufficient cues to determine the purpose. An example is the 'X' often used in the top right corner of pop-up divs (light boxes) to indicate the control for closing the div.

In this case, the label will define a name for the next content inside the div, the payment methods accepted by this store. Does it make sense for you? 

If no one disagrees with, @saranicoly, can you add the `htmlFor` and `id` on the label and div respectively?

Reply:
Sure, I'll add `htmlFor` and `id`. Could you please explain your suggestion above?